### DISCUSSION TITLE
G6安装依赖问题，运行编译时，控制台报错，依赖的ts文件找不到

### DISCUSSION THREAD

**Discussion by ghost** [antvis/G6#3694] [Q&A]
我的环境是mac m1 + node 17.3.1 + react 18.0.1 + @antv/g6 4.6.4 ,在编译运行时，控制台打印异常信息如下：

<img width="1364" alt="iShot_2022-05-19_17 49 42" src="https://user-images.githubusercontent.com/103547464/169267285-8b412bfa-df92-4c82-ba27-7b3ced5d6b58.png">

<img width="1364" alt="iShot_2022-05-19_18 02 38" src="https://user-images.githubusercontent.com/103547464/169268092-23b78b34-25c0-4cde-90d5-6e0be261b6da.png">

<img width="343" alt="iShot_2022-05-19_18 05 52" src="https://user-images.githubusercontent.com/103547464/169268661-c79481f6-9806-4f0d-bd80-c00145391691.png">

4.6.4版本的G6之前在windows上也在用，到没有遇到这个问题，有人遇到过类似的问题吗？

**Comment by Yanyan-Wang** [comment]
看报错路径从 graphin 报出来，去 graphin 仓库问下？
